src/hooks/statsContext.js

```diff
@@ -1,22 +1,89 @@
 import React, { useContext } from 'react';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import Spinner from '../components/Spinner';
 import useFetch from './useFetch';
+
 const AIRTABLE_API_KEY = process.env.REACT_APP_AIRTABLE_API_KEY;
 
 const StatsContext = React.createContext([]);
+const StatusPanel = styled.div`
+  height: 100%;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
 
 const StatsProvider = ({ children }) => {
-  const { data, isLoading, error } = useFetch(
+  const { data, isLoading, isError } = useFetch(
     {
       url:
         'https://api.airtable.com/v0/appo8EvYg4prTdP9h/Table%201?view=Grid%20view',
```

Comment:
What about adding this as a env variable?

Reply:
Done